extensions-core/kafka-indexing-service/src/test/java/org/apache/druid/indexing/kafka/KafkaIndexTaskTest.java

```diff
@@ -2911,7 +2911,8 @@ private KafkaIndexTask createTask(
         maxParseExceptions,
         maxSavedParseExceptions,
         null,
-        null
+        null,
+        false
```

Comment:
Nit: Pass `null` instead so that we fallback to the default, even if it changes in the future.